avaframe/com1DFA/com1DFACfg.ini

```diff
@@ -48,6 +48,9 @@ bufferZoneFactor = 4
 # in addition to the actual release line initialize additionallyFixedFactor*sphKernelRadius*bufferZoneFactor
 # as particles that are not fixed - so that fixed boundaries do not affect the interior of the release particles
 additionallyFixedFactor = 0.275
+# for resetting the mass when relThField is provided, if should be interpolated from release thickness set to true
+# if not mass is computed from massCell/nPartPerCell
```

Comment:
not sure if this explanation is very clear...

Reply:
After the initialization step, the mass of the particles is reset:
- fdOptionIni = False: using the mass of each cell and particles within the cell (mPart = massCell/nPartPerCell) 
-  fdOptionIni = True: by first computing the flow thickness hPart of the particles (bilinear interpolation on the relThField raster) and then mPart = hPart*aPart*rho (here aPart  = aCell/nPartPerCell)
I suggest to use thickness, since I think JT would like to replace depth by thickness

Reply:
I added this, slightly modified